.github/workflows/discord_wiki.yaml

```diff
@@ -0,0 +1,15 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Wiki Changed Discord Notification
+
+on: gollum
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 'oznu/gh-wiki-edit-discord-notification@v1.0.0'
+        with:
+          discord-webhook-url: ${{ secrets.DISCORD_WEBHOOK_WIKI_EDIT }}
```

Comment:
Does it make sense to just have a single DISCORD_WEBHOOK secret that we can reuse for any other notifications?

Reply:
IIUC hooks are linked to specific channels: given wiki cannot get notifications otherwise, maybe better to split so that there's not a choice between either nothing or a everything.

Reply:
That is, with PRs, issues, or discussions, that's a matter of preference. I might prefer GH notifications. Wiki it's not a matter of preference. I wish I could use GH notifications but can't.

Reply:
I didn't realize the secret was tied to a single channel. That makes more sense for having them broken out.